### PR TITLE
front-end Changes

### PR DESCRIPTION
**Pull request by GITANG1**
- changed theme
 - added search by meal type functionality
 - all images are now in the images folder under /assets
 - made register and login component homogeneous
 - fixed page navigation overlapping issue
 - displaying results on home page based on location.

### CODE REVIEW

File: src/app/login/login.component.ts [hunk 1]
```diff
@@ -30,15 +30,16 @@ export class LoginComponent implements OnInit {
     this.authService.authenticateUser(user).subscribe(data => {
       if ( data.success ) {
         this.authService.storeUserData(data.token, data.user);
-        this.flashMessage.show('You are now logged in', {
-          cssClass: 'alert-success',
-          timeout: 5000});
+        // this.flashMessage.show('You are now logged in', {
+        //   cssClass: 'alert-success',
+        //   timeout: 5000});
         this.router.navigate(['home']);
```
Comment (srishtihunjan) [inline src/app/login/login.component.ts]: you can remove the commented lines 33-35.

File: src/app/login/login.component.ts [hunk 2]
```diff
@@ -30,15 +30,16 @@ export class LoginComponent implements OnInit {
...
       } else {
         this.flashMessage.show(data.msg, {
           cssClass: 'alert-danger',
           timeout: 5000});
         this.router.navigate(['login']);
+
```
Comment (srishtihunjan) [inline src/app/login/login.component.ts:39]: remove extra whitespace

File: src/app/register/register.component.ts [hunk 3]
```diff
@@ -3,8 +3,19 @@ import { ValidateService } from '../services/validate.service';
...
+import { MatToolbarModule, MatFormFieldModule, MatCardModule,MatInputModule } from '@angular/material';
 import { FormControl, Validators } from '@angular/forms';
+import { NgModule } from '@angular/core';
+
+
+@NgModule({
+  imports: [
+    MatToolbarModule, MatFormFieldModule, MatCardModule,MatInputModule ,
+    FormControl      //<----------make sure you have added this.
+  ],
```
Comment (srishtihunjan) [inline src/app/register/register.component.ts]: I dont think we need this comment "make sure you have added this"

**Review by srishtihunjan — CHANGES_REQUESTED** [review]
I've mentioned  the changes inline

**Review by srishtihunjan — APPROVED** [review]

**Review by nakhe93 — APPROVED** [review]
Looks good

**Pull request merged**